### PR TITLE
fix(multiple): text input not resized on window resize

### PR DESCRIPTION
**Pull request by romario333**
Text input within multiple select is not resized correctly on window resize. Open this plunkr and try to resize window. Notice that text input (with green background) is not resized:

http://embed.plnkr.co/cqsb8nOIunj2rMU36z5H/preview

Here's the demo with this fix applied: http://embed.plnkr.co/pPdeELl80hgCOVZKCs6a/preview

### CODE REVIEW

**Comment by dangoodman** [comment]
I have just made a PR fixing the same issue, just with a more general approach — https://github.com/angular-ui/ui-select/pull/1252 .

File: src/uiSelectMultipleDirective.js [hunk 1]
```diff
@@ -399,6 +399,14 @@ uis.directive('uiSelectMultiple', ['uiSelectMinErr','$timeout', function(uiSelec
         });
       });
 
+      function onWindowResize() {
+        $select.sizeSearchInput();
+      }
+
+      angular.element($window).on('resize', onWindowResize);
```
Comment (wesleycho) [inline src/uiSelectMultipleDirective.js]: This needs to be debounced.

**Comment by wesleycho** [comment]
This needs tests.

**Comment by user378230** [comment]
This appears to have been implemented somewhere else and subsequently improved on by #1536 and #1538. 

Closing.

**Pull request closed without merging**